tests/MageTest/Command/AbstractCommandTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace MageTest\Command;
+use Mage\Command\AbstractCommand;
+use MageTest\TestHelper\BaseTest;
+use PHPUnit_Framework_MockObject_MockObject;
+
+/**
+ * Class AbstractCommandTest
+ * @package MageTest\Command
+ * @author Jakub Turek <ja@kubaturek.pl>
+ * @coversDefaultClass Mage\Command\AbstractCommand
+ */
+class AbstractCommandTest extends BaseTest
+{
+    /**
+     * @var AbstractCommand|PHPUnit_Framework_MockObject_MockObject
+     */
+    private $abstractCommand;
+
+    /**
+     * @before
+     */
+    public function before()
+    {
+        $this->abstractCommand = $this->getMockForAbstractClass('Mage\Command\AbstractCommand');
+    }
+
+    /**
+     * @covers ::setConfig
+     */
+    public function testSetConfig()
+    {
+        $configMock = $this->getMock('Mage\Config');
+        $this->abstractCommand->setConfig($configMock);
+
+        $actual = $this->getPropertyValue($this->abstractCommand, 'config');
+        $this->assertEquals($configMock, $actual);
+    }
+
+    /**
+     * @covers ::getConfig
+     */
+    public function testGetConfig()
```

Comment:
I agree, I wanted just test the current code with almost no changes. I also agree that dependency injection would help us a lot but for now I wanted only to test if the classes are constructed properly, nothing else. i think we shouldn't do any refactor till we test the weakest points of the project.